README.md

```diff
@@ -126,6 +187,12 @@ excluded because for:
 ## Link checking as part of Github workflow
 
 We configured a [Github workflow](../.github/workflows/check-links.yml) to build the Gatsby site in
-production as a docker image and push it to Docker hub at [`jimmychu0807/substrate-docs`](https://hub.docker.com/repository/docker/jimmychu0807/substrate-docs).
+production as a docker image and push it to Docker hub at
+[`jimmychu0807/substrate-docs`](https://hub.docker.com/repository/docker/jimmychu0807/substrate-docs).
```

Comment:
@jimmychu0807 is this still correct? can we publish under a formal devhub account?

Reply:
NOTE: this should not be a blocker for this PR, IMHO